docs/6-developer-guide/2-testing.md

```diff
@@ -89,25 +91,25 @@ go test ./src/pkg/... -v
 go test ./src/pkg/... -v -run TestFooBarBaz
 ```
 
-### Adding More CLI Unit Tests
+### Adding New CLI Unit Tests
 
-There are a few requirements to be considered when thinking about adding new unit tests.
+When adding new unit tests, please ensure that the following requirements are met:
 
-1. Is what I want to test a true unit (i.e. a single function or file)?
-2. Does what I want to test have a clearly defined interface (i.e. a public specification)?
-3. Is this code inside of the `src/pkg` folder or should it be?
+1. The test must focus on a true unit, such as a single function or file.
+2. The code being tested must have a clearly defined interface, such as a public specification.
+3. The code being tested should be located within the `src/pkg`.
 
-If the answer to these is yes, then this would be a great place for a unit test, if not, you should likely consider writing an end-to-end test instead, or modifying your approach so that you can answer yes.
+If all these requirements are met, then a unit test would be appropriate. If not, please consider writing an end-to-end test instead or modify your approach to meet these requirements.
 
-To create a unit test, look for or add a file ending in `_test.go` to the package for the file you are looking to test (e.g. `auth.go` -> `auth_test.go`).  Import the testing library and then create your test functions as needed.  If you need to mock something out consider the best way to do this, and if it is something that can be used in many tests, consider placing the mock in `./src/test/mocks/`.
+To generate a unit test, search for or include a file that ends with `_test.go` to the package for the file that requires testing, such as `auth.go` -> `auth_test.go`. Import the testing library and create test functions as necessary. In case you need to mock something out, determine the most suitable approach and if the mock can be utilized in multiple tests, consider placing it in  `./src/test/mocks/`. This will help enhance the efficiency and organization of your unit tests.
```

Comment:
```suggestion
To create a unit test, search for or create a file that ends with `_test.go` in the package of the file that requires testing, such as `auth.go` -> `auth_test.go`. Import the testing library and create test functions as necessary. In case you need to mock something out, determine the most suitable approach and if the mock can be used in multiple tests, consider placing it in  `./src/test/mocks/`. This will help enhance the efficiency and organization of the unit tests.
```